src/main/webapp/WEB-INF/jsp/shoppingcart.jsp

```diff
@@ -624,7 +624,7 @@
     function doCheckout() {
         var options = {
             endpoint: './transaction-create.html',
-            commerceLogo: 'images/icons/logo-01.png',
+            commerceLogo: 'http://18.216.122.230/modal/images/icons/logo-01.png',
```

Comment:
This hardcoded IP looks very bad. I think you will have to (ab)use rawgit again and point to the logo as it's on this github repo.

Reply:
You right again. I will change it now.

Reply:
Done